### DISCUSSION TITLE
Is it possible to set permissions on the webview iframe to enable xr-spatial-tracking?

### DISCUSSION THREAD

**Discussion by itsdouges** [microsoft/vscode-discussions#2385] [Extension Development QnA]
Howdy folks! I've been developing a VS Code extension called Triplex for VS Code. I've been exploring a WebXR simulation feature however it looks like I'm currently blocked by the parent webview iframe not having the required permissions:

What it has set:

```
allow="cross-origin-isolated; autoplay; clipboard-read; clipboard-write;"
```

What I need it to be set with:

```
allow="cross-origin-isolated; xr-spatial-tracking; accelerometer; gyroscope; display-capture; geolocation; camera; microphone;"
```

Is it possible to change what's set in the allow attribute when creating a webview / custom editor?

**Comment by robconery** [comment]
@isidorn can you help with this?

**Reply by isidorn** [reply]
It is not possible to change the iframe permissions due to security reasons.
@mjbvz can correct me if needed.
